fiatconnect-api.md

```diff
@@ -2213,6 +2214,23 @@ platform can be used to consume Fiat Connect services by providing their `mobile
 }
 ```
 
+####  8.3.2.4. `BankAccount`
+
+`BankAccount` is a Fiat Account schema that tends to come up with a bank account representation we can use regardless of the country. Meanwhile, the most important field there is the `iban` that stands for International Bank Account Number. It respects a specific organization that you can find here [**IBAN on WIKIPEDIA**](https://fr.wikipedia.org/wiki/International_Bank_Account_Number) and [**IBAN**](https://www.pricebank.fr/questions-reponses/questions-frequentes/qu-est-ce-que-le-code-iban.html).
+
+```
+{
+	accountName: `string`,
+	institutionName: `string`,
+	iban: `string`,
+  bankName?: `string`,
+	country: `string`,
+	fiatAccountType: `FiatAccountTypeEnum.BankAccount`
+}
+```
+
+The `country` field should be a [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code. `bankName` was added for verification purpose only (In a case where there maybe a mistake as each bank located in a specific country can only have a unique international identifier).
```

Comment:
```suggestion
The `country` field MUST be a [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code. `bankName` was added for verification purpose only (In a case where there maybe a mistake as each bank located in a specific country can only have a unique international identifier).
```

Reply:
there is a logical difference between "should" and "must" to look out for here

Reply:
Also, what exactly is `bankName` supposed to be? Wouldn't it have to be a machine-readable value to be useful? (e.g. one of a static set of expected values that a CICO provider's server knows what to do with.) And why is it optional? 

If it's just redundant with `iban` because the bank can be inferred from the `iban` number, I vote we leave it out. The client (e.g. a wallet like Valora) can always perform checks like showing the user a logo of the bank associated with that `iban` (assuming this mapping is easy to do). 

Otherwise, we need an enum of valid bank names to be added (see section 8.1 for examples of this).

Reply:
`bankName` was added for a verification purpose as the IBANNumber contains the Bank code. This is why I did not make it required. But as you stated, the best approach is to remove it and keep only the `iban`.